nix/configuration.nix

```diff
@@ -43,6 +41,8 @@
     defaults.email = config.jl4-demo.acme-email;
   };
 
+  security.sudo.wheelNeedsPassword = false;
```

Comment:
nit: many people expect that when they don't log in as root, that they're protected from doing something really bad by a password

Reply:
i'm hoping those people will not be logging in to our very restricted deployment environment :D